nixos/modules/services/misc/blenderfarm.nix

```diff
@@ -0,0 +1,141 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  cfg = config.services.blenderfarm;
+  json = pkgs.formats.json { };
+  configFile = json.generate "ServerSettings" (defaultConfig // cfg.ServerConfig);
+  defaultConfig = {
```

Comment:
Is this actually necessary? I'm pretty sure you can just supply `{}` as a default

Reply:
Blenderfarm expects this exact default config to be present